nautilus_trader/accounting/manager.pyx

```diff
@@ -262,18 +253,26 @@ cdef class AccountsManager:
 
                 # Apply base xrate
                 locked = round(locked * base_xrate, currency.get_precision())
+                balance_locked = Money(locked, currency)
 
-            total_locked += locked
+            if currency in total_locked:
+                total_locked[currency] = Money(round(total_locked[currency].as_decimal() + balance_locked.as_decimal(), currency.get_precision()), currency)
```

Comment:
There is a more efficient way to do this, but I'll refine on `develop` branch once it lands 👍